include/IFrameBuffer.h

```diff
@@ -50,12 +53,31 @@ class IRenderBuffer;
 class IFrameBuffer : public virtual IReferenceCounted
 {
     public:
+		//! Attaches given texture to given attachment point.
+		/** @param attachmentPoint Identifies attchment point.
+		@param tex Texture being attached.
+		@param mipMapLayer
+		@param layer
```

Comment:
layer of the framebuffer to attach to, for explanation see the OpenGL Wiki for the topic of "layered rendering"
values >= mean that a particular `layer` of a 2D or cubemap texture array, or 3D texture is attached to the FrameBuffer.
value <0 means the entire 3D texture or, 2D texture or cubemap array is bound making the FrameBuffer layered, and enabling you to use gl_Layer for layered rendering.

Look at example 15 to see layered rendering used on a cubemap to render point shadows in a single pass.